src/aggregator/aggregator/elem_test.go

```diff
@@ -3275,5 +3286,9 @@ func verifyOnForwardedFlushResult(t *testing.T, expected, actual []testOnForward
 	require.Equal(t, len(expected), len(actual))
 	for i := 0; i < len(expected); i++ {
 		require.True(t, expected[i].aggregationKey.Equal(actual[i].aggregationKey))
+		require.Equal(t, len(expected[i].expiredTimes), len(actual[i].expiredTimes))
```

Comment:
👍

Reply:
and forgot to mention, this assertion would have exposed the bug (tests failed without the code change).